proj2-feedr/js/app2.js

```diff
@@ -0,0 +1,66 @@
+/* Feedr */
+
+// THE GUARDIAN API INFO
+// put this in an object named API sources. The Guardian is the key with sub-object with this info:
+var root = "https://content.guardianapis.com/search?section=technology&show-fields=body,thumbnail,wordcount&api-key=";
+var key = "830e55a9-fe2e-4cb2-9de4-0ba66be99e23";
+var section = "technology";
+
+// Global vars
+var articleResults;
+
+$.ajax({
+  url: `${root}${key}`,
+	method: "GET"
+}).then(function(data) {
+	console.log(data);
+  addContentToDom(data);
+  articleResults = data.response.results;
+});
+
+function addContentToDom(data) {
+  // response object. results is an object with 10 articles
+  var results = data.response.results;
+  var articleElement = $(".article");
+
+  // loop through each article element and insert content
+  for(var i=0; i < articleElement.length; i++) {
+    // add a data-article attribute with the index of the article
+    var articleDataAttr = $(articleElement[i]).attr('data-article', `${i}`);
+    // DOM elements to attach content to
+    var articleTitle = $(articleElement[i]).find("h3")[0];
+    var categoryLabel = $(articleElement[i]).find("h6")[0];
+    var articleThumb = $(articleElement[i]).find("img")[0];
+    var articleWordCount = $(articleElement[i]).find(".impressions")[0];
+    articleTitle.innerText = results[i].webTitle;
+    categoryLabel.innerText = results[i].sectionName;
+    articleWordCount.innerText = `Word count: ${results[i].fields.wordcount}`;
+    // add article thumbnail
+    $(articleThumb).attr("src", results[i].fields.thumbnail);
+  }
+}
+
+$(".articleContent h3").on("click", function displayArticle() {
```

Comment:
Since we're already using jQuery here, let's consider how jQuery easily handles event delegation: https://learn.jquery.com/events/event-delegation/

We can simplify the code in this click function by using delegation to find our `this` and then work from there.

Reply:
Thanks for the feedback @sonylnagale !